src/main.cpp

```diff
@@ -204,6 +207,10 @@ int main(int argc, char **argv) {
     percents = getenv("WAKE_PERCENT");
   }
 
+  if (strict) {
+    setenv("WAKE_STRICT_PATH", "y", 1);
```

Comment:
I think elsewhere Wake uses `"1"` and `"0"` to indicate boolean values for environment variables (`USE_FUSE_WAKE` is the one I'm thinking of), so we'd probably want to be consistent with that.